hack/make-rules/verify.sh

```diff
@@ -80,6 +80,7 @@ QUICK_PATTERNS+=(
   "verify-vendor-licenses.sh"
   "verify-gofmt.sh"
```

Comment:
it would be more friendly if the tool could fix the import order rather than failing in CI and making the contributor compare failure output to source. the equivalent of `hack/update-gofmt.sh` seems lower friction to me

Reply:
i think having a `hack/update-imports-order.sh` makes sense. this means the tool must also be able to do in-place corrections to the inspected files, @xinydev 

but we still have to run `hack/verify-imports-order.sh` in CI, similarly to how we run `hack/verify-gofmt.sh`.

Reply:
sure, but I wouldn't add the CI blocker until there's an "easy" way for someone to resolve the issues (I'd even consider building it into update-gofmt since people are used to running that)

Reply:
@liggitt this linters is ONLY for cmd/kubeadm as proposed in the PR :)

Reply:
> sure, but I wouldn't add the CI blocker until there's an "easy" way for someone to resolve the issues

that's fair.

> I'd even consider building it into update-gofmt since people are used to running that

as a contributor, this SGTM as well. @spiffxp would you agree to include this under update-gofmt?

Reply:
@dims

> @liggitt this linters is ONLY for cmd/kubeadm as proposed in the PR :)

well that is true, but @liggitt 's suggestions are good because they establish easier adoption for the wider k/k eventually.

Reply:
@neolit123 totally agree.